doc/rest_api/snippet_sharing/use_cases/UC_details_snippet.txt

```diff
@@ -0,0 +1,17 @@
+Summary:
+----------------------------
+Title: viel details for specific configuration snippet
+Scope: Detailed View
+Level: User Goal
+Actors: Anonymous User (+ all others)
+Brief: Any user can view all details about an entry (configuration snippet).
+
+Scenarios:
+----------------------------
+Preconditions: URI of a snippet entry is known
+Main success scenario: User retrieves a response containing all information about the entry, including tags and other meta information like a description, as well as the configuration snippet converted into all supported formats.

```

Comment:
Does the use case really have to contain all detailed information already? What exactly the response contains is specified in the API description anyway. An example what meta information can mean is already given...